base/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,7 @@
     <string name="through">Through %s</string>
     <string name="infoPrompt">Access your memeber card, see museum hours, or update your app settings.</string>
     <string name="accessMemberCard">Access Memeber Card</string>
+    <string name="channel_name">art institute of chicago</string>
```

Comment:
the channel name should be descriptive of the notification. In this case its "audio"